### PR TITLE
Inline Project creation

### LINKED ISSUE #77: Adding new project, in "My Staffplan" or in a client project screen, should open a blank project row inline (not a plop-up)
image

### PR DESCRIPTION
**Pull request by HankC138**
A project can now be created for a client within the mystaffplan page and it will be automatically assigned to the staffplan user.  Closes #77

### CODE REVIEW

File: app/components/userAssignment/tempProjectLabel.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+	setTempProjectOpen: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+export const TempProjectLabel = ({
+	assignment,
+	tempProjectOpen,
+	setTempProjectOpen,
+}: TempLabelProps) => {
+	const {userId} = useParams();
+	console.log(assignment, "ASSIGNEMNT IN TEMP");
```
Comment (fermion) [inline app/components/userAssignment/tempProjectLabel.tsx:22]: Should we remove this and other `console.log` statements?

File: app/people/[userId]/page.tsx [hunk 2]
```diff
@@ -46,35 +47,92 @@ const UserPage: React.FC = () => {
...
-	}, [clientSide, userList, params.name, viewsFilter]);
+	}, [clientSide, userList, params.userId, viewsFilter]);
 
 	if (!userList) return <LoadingSpinner />;
 	const onClose = () => setAddAssignmentVisible(false);
 	const onComplete = () => {
 		setAddAssignmentVisible(false);
 	};
+
+	// const addNewAssignment = (newAssignment: AssignmentType) => {
```
Comment (fermion) [inline app/people/[userId]/page.tsx:65]: Delete?

**Review by fermion — COMMENTED** [review]
Gave it a quick once over. Would defer to @MyStarrySpace and @zhukovdigital, looks like a stray console log and some dead code that could be removed 🙏🏻

File: app/components/userAssignment/tempProjectLabel.tsx (hunk 1 above)
Comment (HankC138) [reply app/components/userAssignment/tempProjectLabel.tsx:22]: Absolutely will remove these

File: app/people/[userId]/page.tsx (hunk 2 above)
Comment (HankC138) [reply app/people/[userId]/page.tsx:65]: removed

**Review by fermion — APPROVED** [review]

**Pull request merged**